sdk/storage/azure-storage-queue/samples/queue_samples_message_async.py

```diff
@@ -279,6 +279,49 @@ async def update_message_async(self):
             finally:
                 # Delete the queue
                 await queue.delete_queue()
+    
+    async def receive_messages_with_max_messages(self):
+        # Instantiate a queue client
+        from azure.storage.queue.aio import QueueClient
+        queue = QueueClient.from_connection_string(self.connection_string, "myqueue7")
+
+        # Create the queue
+        async with queue:
+            await queue.create_queue()
+
+            try:
+            # [START send_messages]
```

Comment:
I don't think we need the snippet tags here, unless you actually plan to add these into the reference documentation?